bpf/go_nethttp.c

```diff
@@ -147,7 +147,7 @@ int uprobe_readRequestStart(struct pt_regs *ctx) {
             void *tls_state = 0;
             bpf_probe_read(&tls_state, sizeof(tls_state), (void *)(c_ptr + c_tls_pos));
             conn_conn_ptr = unwrap_tls_conn_info(conn_conn_ptr, tls_state);
-            bpf_dbg_printk("conn_conn_ptr %llx, tls_state %llx, c_tls_pos = %d, c_tls_ptr = %llx", conn_conn_ptr, tls_state, c_tls_pos, c_ptr + c_tls_pos);
+            //bpf_dbg_printk("conn_conn_ptr %llx, tls_state %llx, c_tls_pos = %d, c_tls_ptr = %llx", conn_conn_ptr, tls_state, c_tls_pos, c_ptr + c_tls_pos);
```

Comment:
Are the comments made on purpose?

Reply:
Yes, I haven't been able to figure out why, but on kernel 5.10, some of our bpf_printk are not accepted. It reports unknown function... :(. It typically happens if there's a printk in a loop, but some of these that I had to comment weren't in loops. It won't affect users if any slips, only shows if we have BPF_DBG on.